tests/conftest.py

```diff
@@ -12,3 +21,240 @@ def api_test_config() -> APITestSessionConfig:
 
     """
     return APITestSessionConfig()
+
+
+def get_env(variable_name: str) -> str:
+    """Returns an environment variable"""
+    try:
+        var = os.environ[variable_name]
+        if not var:
+            raise RuntimeError(f"Variable is null, Check {variable_name}.")
+        return var
+    except KeyError:
+        raise RuntimeError(f"Variable is not set, Check {variable_name}.")
+
+
+def get_env_file(variable_name: str) -> str:
+    """Returns an environment variable as path"""
+    try:
+        path = os.path.abspath(os.environ[variable_name])
```

Comment:
Could this use the `get_env` method created above? I think it would result in something like
```
path = os.path.abspath(get_env(variable_name))
```